frontend/src/lib/uPlotLib/plugins/tooltipPlugin.ts

```diff
@@ -256,6 +259,15 @@ const tooltipPlugin = ({
 
 	const apiResult = apiResponse?.data?.result || [];
 
+	let customText: HTMLDivElement;
+	if (isMessagingQueueCustomTooltipText) {
+		customText = document.createElement('div');
+		customText.textContent = 'Click on co-ordinate to view details';
```

Comment:
rather than passing the boolean and constructing the customText element here ? can we just pass this from consumer? will make it generic so that anyone can use it

Reply:
done - made `customTooltipElement?: HTMLDivElement`